iroha-cli/interactive/impl/interactive_common_cli.cpp

```diff
@@ -107,9 +127,8 @@ namespace iroha_cli {
     }
 
     boost::optional<std::vector<std::string>> parseParams(
-        std::string line, std::string command_name, ParamsMap params_map) {
-      auto params_description =
-          findInHandlerMap(command_name, std::move(params_map));
+        std::string line, std::string command_name, ParamsMap &params_map) {
```

Comment:
I guess it should be `const ParamsMap & `

Reply:
Actually not, it is intentional it to be not `const ParamsMap &` since it gets updated with user input values.

Reply:
Thanks. Missed that